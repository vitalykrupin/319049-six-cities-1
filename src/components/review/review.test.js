import React from 'react';
import renderer from 'react-test-renderer';
import Review from '../review/review';

describe(`Review`, () => {
  const review = {
    id: 2,
    comment: `Weird place`,
    rating: 1.5,
    date: `2019-05-16T21:02:58.227Z`,
    user: {
      avatarUrl: `path.jpg`,
      id: 8,
      isPro: false,
      name: `Kurt`,
    },
  };

  it(`renders correctly`, () => {
    const tree = renderer
      .create(<Review review={review}/>)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
