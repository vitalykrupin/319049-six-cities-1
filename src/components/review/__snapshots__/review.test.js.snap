// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Review renders correctly 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="path.jpg"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Kurt
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "30%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      Weird place
    </p>
    <time
      className="reviews__time"
      dateTime="2019-05-16T21:02:58.227Z"
    >
      May
       
      2019
    </time>
  </div>
</li>
`;
